docs/testing/releases/490.md

```diff
@@ -0,0 +1,109 @@
+## Testing notes and ZIP for release 4.9.0
+
+Zip file for testing: [woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce-gutenberg-products-block/files/6295807/woocommerce-gutenberg-products-block.zip)
+
+## Feature plugin and package inclusion in WooCommerce core
+
+### Remove deprecated shortcode save handling from SSR Blocks (#4010)
+
+Smoke test these blocks can be inserted and render correctly:
+
+* Hand-Picked products Block
+* Best Selling Products Block
+* Top Rated Products Block
+* Newest Products Block
+* On Sale Products Block
+* Products by Category Block
+* Products by Tag Block
+* Products by Attribute Block
+
+### Fix Featured Product and Featured Category button alignment (#4028)
+
+1. Add a Featured Product and Featured Category blocks.
+2. Without making any change on them, preview them in the frontend and verify the button is centered.
+3. Change the alignment of the button to the left, right, back to the center, etc. and verify it works.
+
+| Before | After |
+| --- | --- |
+| ![imatge](https://user-images.githubusercontent.com/3616980/113333925-5985a000-9323-11eb-8c11-25796187bbcc.png) | ![imatge](https://user-images.githubusercontent.com/3616980/113333839-3e1a9500-9323-11eb-9e12-4bd783487638.png) |
+
+### Enable Google Analytics Integration (#4020)
```

Comment:
Some of these testing steps apply to the feature plugin only, especially I think around "cart" and "checkout" events? @mikejolley can you also confirm:

- Is there a chance that existing event tracking by Google Analytics users could be affected - and does that need verified to not be impacted?
- For the testing steps regarding cart and checkout - I'm assuming that's _just_ the Cart and Checkout block flows right?
- Is it expected that event tracking added in blocks that are surfaced in Woo Core should work even if Cart and Checkout related code is not active in that environment?

Depending on Mike's answers here Albert, we may need to update the testing steps to make it more clear for when these changes are surfaced for future Woo Core package updates.

Reply:
@nerrad there is no longer a flag, so these changes will affect all builds, the prerequisite being that `WC_Google_Analytics_Integration` exists. So testing anything from this integration requires the free GA extension active also. The event tracking code is blocks specific so I don't see it impacting existing events if blocks are not in use.

Reply:
> Depending on Mike's answers here Albert, we may need to update the testing steps to make it more clear for when these changes are surfaced for future Woo Core package updates.

I will. :+1: 

> For the testing steps regarding cart and checkout - I'm assuming that's just the Cart and Checkout block flows right?

I assumed that too. Cart & Checkout related steps are [marked](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4053/files/6ed1c972d9a3d7472177f37b2816e05132027d3a#diff-de38dd98c08e4f1243f54cc86f83a1d88baf79996c31ac9f2a5cf5614ddc28dbR43) with `(feature plugin only)` in the testing instructions.

Reply:
> I assumed that too. Cart & Checkout related steps are marked with (feature plugin only) in the testing instructions.

Yup I saw that. But if we have testers not from our team assist testing things in the future, do you feel its still clear enough in the steps that we want checkout flows with blocks tested?